src/TiledArray/algebra/diis.h

```diff
@@ -267,6 +321,10 @@ namespace TiledArray {
         }
       }
 
+      EigenVectorX get_coeffs() { return C_; }
+
+      unsigned int get_nskip() { return nskip_; }
```

Comment:
Similarly, assert that C_ is nonempty so that calls to get_coeffs() before calling extrapolate fail. And add dox.